sdk/cognitivelanguage/Azure.AI.Language.Conversations/tests/ConversationAnalysisClientLiveTests.cs

```diff
@@ -20,152 +21,208 @@ public ConversationAnalysisClientLiveTests(bool isAsync, ConversationsClientOpti
+            Assert.AreEqual("Orchestration", conversationalTaskResult.Result.Prediction.ProjectKind?.ToString());
 
             // assert - top intent
             Assert.AreEqual("RestaurantIntent", conversationalTaskResult.Result.Prediction.TopIntent);
 
             // cast prediction
-            var orchestrationPrediction = conversationalTaskResult.Result.Prediction as OrchestrationPrediction;
+            dynamic orchestrationPrediction = conversationalTaskResult.Result.Prediction;
             Assert.IsNotNull(orchestrationPrediction);
 
             // assert - not empty
-            Assert.IsNotEmpty(orchestrationPrediction.Intents);
+            Assert.IsNotEmpty((IEnumerable)orchestrationPrediction.Intents);
 
             // cast top intent
-            var topIntent = orchestrationPrediction.Intents[orchestrationPrediction.TopIntent] as LuisTargetIntentResult;
+            dynamic topIntent = orchestrationPrediction.Intents[(string)orchestrationPrediction.TopIntent];
             Assert.IsNotNull(topIntent);
 
             // assert - inent target kind
-            Assert.AreEqual(TargetProjectKind.Luis, topIntent.TargetProjectKind);
+            Assert.AreEqual("Luis", topIntent.TargetProjectKind?.ToString());
         }
 
         [RecordedTest]
         public async Task AnalyzeConversation_Orchestration_QuestionAnswering()
         {
-            ConversationalTask conversationalTask = new(
-                new ConversationAnalysisOptions(new TextConversationItem("1", "1", "How are you?")),
-                new ConversationTaskParameters(TestEnvironment.OrchestrationProjectName, TestEnvironment.OrchestrationDeploymentName));
+            var data = new
+            {
+                analysisInput = new
+                {
+                    conversationItem = new
+                    {
+                        text = "How are you?",
+                        id = "1",
+                        participantId = "1",
+                    }
+                },
+                parameters = new
+                {
+                    projectNam = TestEnvironment.OrchestrationProjectName,
+                    deploymentName = TestEnvironment.OrchestrationDeploymentName,
+                },
+                kind = "Conversation",
+            };
 
-            Response response = await Client.AnalyzeConversationAsync(conversationalTask.AsRequestContent());
+            Response response = await Client.AnalyzeConversationAsync(RequestContent.Create(data));
 
             // assert - main object
             Assert.IsNotNull(response);
 
             // deserialize
-            using JsonDocument json = await JsonDocument.ParseAsync(response.ContentStream);
-            ConversationalTaskResult conversationalTaskResult = ConversationalTaskResult.DeserializeConversationalTaskResult(json.RootElement);
+            dynamic conversationalTaskResult = response.Content.ToDynamic(DynamicJsonNameMapping.PascalCaseGetters);
```

Comment:
```suggestion
            dynamic conversationalTaskResult = response.Content.ToDynamic(DynamicJsonOptions.AzureDefault);
```

Reply:
I can do this, but I have a question/feedback: if the default - no parameter - is to use the original case, which I fully support because the re-casing is confusing, as we covered offline in the past, why do we have a value with "default" in the name that doesn't actually match the default behavior with no value specified? I get that it's "AzureDefault", but still seems potentially misleading.

Reply:
Regardless of the name, I think the documentation for `AzureDefault` should reflect what it actually does. That's why I used `PascalCaseGetters`: it's explicit and obvoius what it does. I'm not personally a fan of using something that isn't explicit about its intent or is otherwise obvious.

Reply:
I added this to the feedback thread. We can resume there.